### DISCUSSION TITLE
Can backups be scheduled as a task in paperless-ng

### DISCUSSION THREAD

**Discussion by AvdH** [jonaswinkler/paperless-ng#577] [Support] [answered]
I have seen in the docs that you can backup the documents using the document_exporter. 
with docker-compose
`$ docker-compose exec webserver <command> <arguments>`

with docker 
`$ docker exec -it <container-name> <command> <arguments>`

Is it possible to run this command (on a daily basis) as a scheduled task?. If seen other commands like sanity_checker that runs as a scheduled task and can be run manually.

**Comment by jonaswinkler** [comment, marked as answer]
See #508.

However, I'd run the exporter as a cron job on the host system, as part of a script that also does incremental backups with `rsync`.

**Reply by AvdH** [reply]
Thanks a lot. 
This works!

**Reply by AvdH** [reply]
A cron job on the host is not an option for me. I run a docker swarm, so paperless-ng could run on any of those nodes.
I even have some cron-jobs running inside docker (using crazymax/swarm-cronjob)
